pipes/rules/metagenomics.rules

```diff
@@ -85,12 +85,12 @@ rule krona_import_taxonomy:
     input: lambda wildcards: join(config["data_dir"], config["subdirs"]["metagenomics"], \
            ".".join([wildcards.sample, wildcards.adjective, method_props[wildcards.method]['reads_ext']]))
     output: join(config["data_dir"], config["subdirs"]["metagenomics"], "{sample}.{adjective,raw|cleaned}.{method,kraken|diamond|rna_bwa|rna_bwa_dupes}.krona.html")
-    run:
-        # method_props key can be changed to allow for different tools, 
```

Comment:
Let's actually make this deleted comment correct (make the code reflect what it says) instead of deleting it.